### DISCUSSION TITLE
How to add custom template for shortcode？

### DISCUSSION THREAD

**Discussion by licoded** [thuliteio/doks#713] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/42663693/159613198-f0fcf658-90c1-4d32-9a3c-3980bd94590f.png)

**Comment by licoded** [comment, marked as answer]
I have find the answer myself [here](https://github.com/h-enk/doks/tree/master/layouts/shortcodes).

**Reply by h-enk** [reply]
Yes, that's correct. Will be part of the upcoming release v0.4.1 — available soon
